webrtc-stats.html

```diff
@@ -464,6 +465,23 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>remoteTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Present if <code>isRemote</code> is <code>true</code>,
+                  <code>remoteTimestamp</code>, of type
+                  <code>DOMHighResTimeStamp</code> [[!HIGHRES-TIME]],
+                  represents the remote timestamp at which these statistics
```

Comment:
Do we want the wiggle room for timestamps from RTCP RR/XRs? otherwise I would suggest moving this to RTCOutboundRTPStreamStats, i.e., being consistent with the data from the RTCP SRs.